docs/tutorials/MenuGuide.md

```diff
@@ -0,0 +1,237 @@
+
+
+### 3 Add some menu items
+-------------------
+
+Ok, now we've got the Menu, but we need some items, under which you can choose in the menu. You add them underneath the CreateMenu that we made earlier.
+
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "LS");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "LS");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "SF");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "SF");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "LV");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "LV");
+ 
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 1, "Grove Street");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 1, "Starfish Tower");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 1, "Wheel Arch Angels");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 1, "Jizzys");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 1, "4 Dragons");
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 1, "Come-a-Lot");
+
+The explanation for AddMenuItem:
+
+[AddMenuItem](../scripting/functions/AddMenuItem)(menuid, column, text);
+
+|menuid| The menuid of the menu where the item shall be displayed|
+|--|--|
+|column| The column in which the item shall be shown|
+|text|   The text of the item|
+
+
+### 4 Creating the effects for the items
+----------------------------------
+
+Okay, now that we have created a full menu with items what should happen when you choose an item? In our example we want to make a teleportmenu, so we should get teleported to the position we choose. When a player selects an item on a menu the script calls the callback: public [OnPlayerSelectedMenuRow](../scripting/callbacks/OnPlayerSelectedMenuRow)(playerid, row). The best way to do it is to do it with a switch, this is like several if statements to check if a variable is worth certain values. But first we only want these effects for the menu we want so we need to create a variable that holds what menu the player is looking at, this is done with 'GetPlayerMenu':
+
+new Menu:CurrentMenu = [GetPlayerMenu](../scripting/functions/GetPlayerMenu)(playerid);
+
+Now, when somebody selects something on the menu, their menuid will be saved in 'CurrentMenu'.
+
+Now we have to check that the menu they selected on is the menu we want:
+
+
+public public [OnPlayerSelectedMenuRow](../scripting/functions/OnPlayerSelectedMenuRow)(playerid, row)
+{
+	new Menu:CurrentMenu = [GetPlayerMenu](../scripting/functions/GetPlayerMenu)(playerid);
+	if (CurrentMenu == teleportmenu)
+	{
+    	//stuff
+	}
+	return 1;
+}
+
+Now in between these brackets is where the 'switch' is, this checks what item the player selected or 'row' this can be done with if statements checking what row it is, but the switch is a much simpler way of writing it.
```

Comment:
```suggestion
Now in between these brackets is where the 'switch' is, this checks which item the player selected. This can be done with if statements, but the switch is a much simpler way of writing it.
```